src/array_api_stubs/_draft/array_object.py

```diff
@@ -343,9 +341,52 @@ def __dlpack__(
 
         Notes
         -----
+        Major DLPack versions represent ABI breaks, minor versions represent
```

Comment:
Not sure if dlpack currently explicitly adopts semantic versioning. It would be good to make it clear there too to avoid mismatch on versioning expectation.

Reply:
Yes they do.